numpyro/util.py

```diff
@@ -8,7 +8,7 @@
 import re
 
 import numpy as np
-import tqdm
+from tqdm.auto import tqdm
```

Comment:
This completely fixes the issue for notebooks for me; great suggestion @fehiepsi !

Reply:
Haha, this is a bit different from what I suggested. ;) I meant to only invoke `tqdm.auto` when running multiple chains. I would prefer to have the old progress_bar for [reasons here](https://github.com/pyro-ppl/numpyro/pull/813#issuecomment-726487003).

Reply:
Ah oops! Ok I'll have a look at that and change it :)

Reply:
I just changed this in the latest pull request: `tqdm.auto` is now only used for `num_chains>1`. For `num_chains=1`: the usual tqdm progress bar is used (on the non-compiled sampler).

Is this what you meant?